src/server/util/readingsUtils.js

```diff
@@ -98,6 +98,21 @@ function expectMaxMinToEqualExpected(res, expected, id = METER_ID) {
     }
 }
 
+/**
+ * Compares readings from api call against the expected readings csv
```

Comment:
I realize other ones don't do this but as we add more expected fuctions it would be good to have the description be more precise. Also, this does not use a CSV so remove that. Thus, could this become: 
Compares readings from compare api call against the expected readings